.github/workflows/debian-draft-executable-release.yml

```diff
@@ -19,6 +19,9 @@ jobs:
       - run: |
```

Comment:
Why we renamed only the debian file?

Reply:
[platform-executables-acceptance-tests.yml](https://github.com/twilio/twilio-cli/pull/374/files#diff-2620ef6cacbc37efec19f000b7c02c86db2ee49aa70297c837ad874d04104b98)  and [windows-executable-release.yml](https://github.com/twilio/twilio-cli/pull/374/files#diff-2a58c3acab590ab41ebd782b3f7e1fec3a87b5edd4b7f408cd1a58e51ffc4db7) would also need renaming, I was going to rename them as a part of cleanup, but will get this done for better.